ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -325,18 +325,18 @@ static inline int end_atomicity(
 }
 
 static inline int get_dynamic_win_info(uint64_t remote_addr, ompi_osc_ucx_module_t *module,
-                                       int target) {
+                                       int target, int *dynamic_win_index) {
     uint64_t remote_state_addr = (module->state_addrs)[target] + OSC_UCX_STATE_DYNAMIC_WIN_CNT_OFFSET;
-    size_t len = sizeof(uint64_t) + sizeof(ompi_osc_dynamic_win_info_t) * OMPI_OSC_UCX_ATTACH_MAX;
-    char *temp_buf = malloc(len);
+    size_t remote_state_addr_len = sizeof(uint64_t) + sizeof(ompi_osc_dynamic_win_info_t) * OMPI_OSC_UCX_ATTACH_MAX;
```

Comment:
Also IMO and can be ignored.
BTW, this is not jsut address, its more a state, right?
```suggestion
    size_t rem_state_len = sizeof(uint64_t) + sizeof(ompi_osc_dynamic_win_info_t) * OMPI_OSC_UCX_ATTACH_MAX;
```

Reply:
but in general, it's just the var name change and not more than that

Reply:
you are correct, not just the length of the address